packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -251,3 +251,86 @@ class VelocityTracker {
     return Velocity(pixelsPerSecond: estimate.pixelsPerSecond);
   }
 }
+
+/// A [VelocityTracker] subclass that matches iOS scroll view's fling velocity
+/// calculation.
+///
+/// To obtain a velocity, call [getVelocity] or [getVelocityEstimate]. The
+/// esimated velocity is typically used as the initial flinging velocity of a
+/// `Scrollable`, when its drag gesture ends.
+class FreeScrollStartVelocityTracker extends VelocityTracker {
```

Comment:
Why this name? Either somehow describe the "free scroll start" nature of the velocity tracking in the class doc or just call this ios velocity tracker.

Reply:
I thought we wanted to avoid having iOS in the name? Would be nice to know that I was mistaken.